### DISCUSSION TITLE
Issue polling foreach

### DISCUSSION THREAD

**Discussion by oscargh0901** [livewire/livewire#5350] [1. Help]
This is a livechat web app, I need to "refresh" when sending a message, poll works well when using now() outside the foreach (to comprobate that works), but inside the foreach the content and now() does nothing!! I need to refresh the browser to see the changes (the new message) and I dont know how to fix it...
![imagen](https://user-images.githubusercontent.com/57479963/200413896-04e0edb5-26da-42b7-adbc-8fc7d49e189a.png)

**Comment by joshhanley** [comment]
@oscargh0901 try adding `wire:key` to any elements inside a loop. See here for details https://laravel-livewire.com/docs/2.x/troubleshooting#dom-diffing-cures

Hope this helps!
